### PR TITLE
Review chapter "Java TCP programming"

### PR DESCRIPTION
**Pull request by ludelafo**

### CODE REVIEW

File: 12-java-tcp-programming/COURSE_MATERIAL.md [hunk 1]
```diff
@@ -265,258 +260,134 @@ for (int i = 0; i < dataLength; i++) {
...
-work.
+Compare your solutions with the official ones stated in the
+[Solution](#solution) section.
 
-You can compare your solution with the official one stated in the
-[Solution](#solution) section, however, **we highly recommend you to try to
-complete the practical content by yourself first to learn the most**.
+If you have any questions about the solution, feel free to ask as described in
+the [Finished? Was it easy? Was it hard?](#finished-was-it-easy-was-it-hard)
+section.
```
Comment (hadrylouis) [inline 12-java-tcp-programming/COURSE_MATERIAL.md:383]: I think it's a good thing to no longer ask the students to create a Github discussion, but simply to create one if needed.

File: 12-java-tcp-programming/COURSE_MATERIAL.md [hunk 2]
```diff
@@ -589,3 +465,7 @@ discuss it!
 
 - Main illustration by [Carl Nenzen Loven](https://unsplash.com/@archduk3) on
   [Unsplash](https://unsplash.com/photos/N8GdKC4Rcvs)
+
+```
+
+```
```
Comment (hadrylouis) [inline 12-java-tcp-programming/COURSE_MATERIAL.md:471]: Remove this

**Review by hadrylouis — CHANGES_REQUESTED** [review]

File: 12-java-tcp-programming/COURSE_MATERIAL.md (hunk 2 above)
Comment (ludelafo) [reply 12-java-tcp-programming/COURSE_MATERIAL.md]: Fixed! Thanks.

**Comment by ludelafo** [comment]
@hadrylouis, @GeraudSilvestri, I have created the template for this chapter (https://github.com/heig-vd-dai-course/heig-vd-dai-course-java-tcp-programming-practical-content-template). For me, the chapter is complete.

**Review by hadrylouis — APPROVED** [review]

**Pull request merged**